project/Dependencies.scala

```diff
@@ -35,7 +35,7 @@ object Dependencies {
   val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.3"
   val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "7.8.6"
   val scalaLogging: ModuleID =   "com.typesafe.scala-logging" %% "scala-logging"   % scalaLoggingV
-  val swaggerUi: ModuleID =      "org.webjars"                %  "swagger-ui"      % "4.1.2"
+  val swaggerUi: ModuleID =      "org.webjars"                %  "swagger-ui"      % "4.5.2"
```

Comment:
looks like the PR that David An linked uses:

```
    "org.webjars.npm"                % "swagger-ui-dist"     % "4.6.1",
```

would we be able to use that here?

Reply:
So, I tried that, and it broke the build because it couldn't find that version. Maybe I need to use the swagger-ui-dist, but I am concerned that it will break in other unpredictable ways.

Reply:
I get the following error messages building in Sam with that:
```
[error] (update) sbt.librarymanagement.ResolveException: Error downloading org.webjars:swagger-ui-dist:4.6.1
[error]   Not found
[error]   Not found
[error]   not found: /Users/srubenstein/.ivy2/local/org.webjars/swagger-ui-dist/4.6.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/webjars/swagger-ui-dist/4.6.1/swagger-ui-dist-4.6.1.pom
[error]   not found: https://artifactory.broadinstitute.org/artifactory/libs-release/org/webjars/swagger-ui-dist/4.6.1/swagger-ui-dist-4.6.1.pom
[error]   not found: https://artifactory.broadinstitute.org/artifactory/libs-snapshot/org/webjars/swagger-ui-dist/4.6.1/swagger-ui-dist-4.6.1.pom
```

Reply:
make sure you are using `"org.webjars.npm"`, not `"org.webjars"`

Reply:
Same problem:
```
[error] (update) sbt.librarymanagement.ResolveException: Error downloading org.webjars.npm:swagger-ui:4.6.1
[error]   Not found
[error]   Not found
[error]   not found: /Users/srubenstein/.ivy2/local/org.webjars.npm/swagger-ui/4.6.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/webjars/npm/swagger-ui/4.6.1/swagger-ui-4.6.1.pom
[error]   not found: https://artifactory.broadinstitute.org/artifactory/libs-release/org/webjars/npm/swagger-ui/4.6.1/swagger-ui-4.6.1.pom
[error]   not found: https://artifactory.broadinstitute.org/artifactory/libs-snapshot/org/webjars/npm/swagger-ui/4.6.1/swagger-ui-4.6.1.pom
[error] (ssExtractDependencies) sbt.librarymanagement.ResolveException: Error downloading org.webjars.npm:swagger-ui:4.6.1
[error]   Not found
```

Reply:
just making sure, you're using   `"org.webjars.npm"` and                `"swagger-ui-dist"` ?

Reply:
I wasn't, that was it.